src/compiler/crystal/command/format.cr

```diff
@@ -11,7 +11,18 @@ class Crystal::Command
     show_backtrace = false
 
     OptionParser.parse(@options) do |opts|
-      opts.banner = "Usage: crystal tool format [options] [file or directory]\n\nOptions:"
+      opts.banner = <<-USAGE
+        Usage: crystal tool format [options] [file or directory ...]
+
+        Formats Crystal code in place.
+
+        If a file or directory is omitted,
+        Crystal source files beneath the working directory are formatted.
+
+        Use '-' in place of a file or directory to format STDIN to STDOUT.
```

Comment:
Maybe mention that multiple file/directory parameters are accepted, but when `-` is used, it must be the only one.